src/libcompiler_builtins/Cargo.toml

```diff
@@ -13,3 +13,4 @@ core = { path = "../libcore" }
 
 [build-dependencies]
 gcc = "0.3.27"
+rustc-cfg = { git = "https://github.com/japaric/rustc-cfg", branch = "llvm-target" }
```

Comment:
I had some comments on the other PR, but I'm not 100% sure that this is buying us enough to pull in a dependency. Many of the `.contains` are now just `relevant_piece == "foo"`, which is good to work over parsed forms but hasn't cause ambiguity problems in the past?